proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,175 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only apply for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF/DC could offer
+at the moment.
+For the free/open online training SCF/DC hopes to satisfy
+the demand for instructor training from their community
+that isn't associate with one partner or affiliate
+and could participate at the Instructor Training
+delivered for partners and affiliates.
```

Comment:
@karinlag What I'm trying to explain here is that we will give preference to people/groups that isn't a member of one partner of affiliate of us.